nym-vpn-app/src-tauri/src/grpc/client.rs

```diff
@@ -188,7 +192,7 @@ impl GrpcClient {
             .listen_to_events(())
             .await
             .inspect_err(|e| {
-                error!("listen_to_tunnel_state_changes failed: {}", e);
+                warn!("listen_to_tunnel_state_changes failed: {}", e); // changed level from error to warning
```

Comment:
Error is, it is a critical issue if this call is failing, for whatever the reason.